
Unfortunately the types also need to be defined when used from user-space
applications, ...

  - Rene Rebe <rene@exactcode.de>

--- linux-2.6.8.1/include/linux/mod_devicetable.h.orig	2004-09-17 13:51:23.577697246 +0200
+++ linux-2.6.8.1/include/linux/mod_devicetable.h	2004-09-17 13:51:30.375605943 +0200
@@ -7,10 +7,8 @@
 #ifndef LINUX_MOD_DEVICETABLE_H
 #define LINUX_MOD_DEVICETABLE_H
 
-#ifdef __KERNEL__
 #include <linux/types.h>
 typedef unsigned long kernel_ulong_t;
-#endif
 
 #define PCI_ANY_ID (~0)
 
